scripts/docker_build/sof_qemu/Dockerfile

```diff
@@ -38,7 +38,9 @@ RUN apt-get -y update && \
 		libpixman-1-dev \
 		pkg-config \
 		sudo \
-		bsdmainutils
+		bsdmainutils \
+		bison \
+		flex
```

Comment:
Nit: backslash the last element + empty line. This makes future diffs look nicer.